<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>160</width>
    <height>144</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>160</width>
      <height>144</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionLeer_ROM"/>
    <addaction name="actionGuardar"/>
    <addaction name="actionCargar"/>
   </widget>
   <widget class="QMenu" name="menuVentana">
    <property name="title">
     <string>Ventana</string>
    </property>
    <addaction name="actionX1"/>
    <addaction name="actionX2"/>
    <addaction name="actionX4"/>
    <addaction name="actionX8"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuVentana"/>
  </widget>
  <action name="actionLeer_ROM">
   <property name="text">
    <string>Leer ROM...</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar...</string>
   </property>
  </action>
  <action name="actionCargar">
   <property name="text">
    <string>Cargar...</string>
   </property>
  </action>
  <action name="actionX1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>x1</string>
   </property>
  </action>
  <action name="actionX2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>x2</string>
   </property>
  </action>
  <action name="actionX4">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>x4</string>
   </property>
  </action>
  <action name="actionX8">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>x8</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
